test/verify/check-realms

```diff
@@ -259,6 +258,16 @@ class TestKerberos(MachineCase):
         self.configure_kerberos()
         self.machine.restart_cockpit()
 
+
+        # These only support the default keytab location
+        keytab = "/etc/cockpit/krb5.keytab"
```

Comment:
We should test as documented. This does not test a symlink (documented above). Suggest changing the documentation, since I don't think a symlink can be made to work without fragile ```chcon``` commands.